### PR TITLE
Add example 03 - covers gRPC compression and max message size

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Purpose
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/2153

This use case is similar to the one used in Choreo.

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: examples/program_analyzer/client/BalProgramAnalyzer_client.bal [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// under the License.
+
+import ballerina/io;
+import ballerina/lang.value;
+
+public function main() returns error? {
+
+    // Change the maximum inbound message size accordingly
+    BalProgramAnalyzerClient ep = check new ("http://localhost:8981", maxInboundMessageSize = 424193);
+    string s = check ep->syntaxTree();
```
Comment (dilanSachi) [inline examples/program_analyzer/client/BalProgramAnalyzer_client.bal:24]: Maybe we can improve the variable names
```suggestion
    string syntaxTreeString = check ep->syntaxTree();
```

**Review by daneshk — DISMISSED** [review]
LGTM

File: examples/program_analyzer/client/BalProgramAnalyzer_client.bal (hunk 1 above)
Comment (BuddhiWathsala) [reply examples/program_analyzer/client/BalProgramAnalyzer_client.bal]: Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/636/commits/b3f4e50a25ee0e28a1ac8d0eaa008b2313e18e2c

**Review by dilanSachi — APPROVED** [review]
LGTM

**Comment by shafreenAnfar** [comment]
LGTM!

**Review by daneshk — APPROVED** [review]

**Pull request merged**